src/Context/WhenStepDefinitions.php

```diff
@@ -35,6 +35,11 @@ public function when_i_launch_in_the_background( $cmd ) {
 	 * @When /^I (run|try) `([^`]+)`$/
 	 */
 	public function when_i_run( $mode, $cmd ) {
```

Comment:
This is not the only step that triggers a WP-CLI command. For proper coverage, we'll need to cover all instances that trigger WP-CLI commands, like (for example) the below `when_i_run_from_a_subfolder()` and `when_i_run_the_previous_command_again()`. There might be others as well...

Reply:
It's only `when_i_run_from_a_subfolder` AFAICT, because `when_i_run_the_previous_command_again` just re-runs the previous command without any substitutions.

I don't think there are any others.

`when_i_launch_in_the_background` is used for launching `wp server` in the background, but it doesn't make sense to invoke coverage collection there. That means no code coverage for server-command but I think we can live with that .